### PR TITLE
Kempe's Algorithm in Python implemented

### PR DESCRIPTION
**Pull request by Riyazul555**
Closes PR  #47

### CODE REVIEW

**Comment by Riyazul555** [comment]
I have resolved all the required changes....please see it once @iuliagroza before I merge it.....Thanks

File: Kempe/Kempe.py
Comment (iuliagroza) [inline Kempe/Kempe.py:5]: The time complexity should be the same no matter the language used for implementation, so maybe O(2^N)?

File: Kempe/Kempe.py
Comment (iuliagroza) [inline Kempe/Kempe.py:6]: Please fix the space complexity as well.

File: Kempe/Kempe.py
Comment (iuliagroza) [inline Kempe/Kempe.py:36]: Nice one!

File: Kempe/Kempe.py
Comment (iuliagroza) [inline Kempe/Kempe.py:68]: I think the modular subtraction formula is (A-B)%MOD = (A%MOD - B%MOD)%MOD, so no need to remove those `MOD`s, but we can remove the added one.

**Review by iuliagroza — APPROVED** [review]
Nice changes! LGTM! You can close the issue after fixing these minor changes.

**Comment by Riyazul555** [comment]
I have fixed the finishing changes also .....merging this pR @iuliagroza

**Pull request merged**
